src/command_modules/azure-cli-interactive/azclishell/app.py

```diff
@@ -562,6 +565,17 @@ def cli_execute(self, cmd):
             CONFIG.load(os.path.join(azure_folder, 'az.json'))
             SESSION.load(os.path.join(azure_folder, 'az.sess'), max_age=3600)
 
+            config = Configuration()
+            self.app.initialize(config)
+            self.app.session = {
+                'headers': {
+                    'x-ms-client-request-id': str(uuid.uuid1())
+                },
+                'command': 'unknown',
```

Comment:
Can we use a better name for the command? We know that it is the "internal" cd command, no?